docs/mobile-apps/automated-testing/appium/appium-versions.md

```diff
@@ -172,7 +182,7 @@ The following list of custom Appium plugins are supported:
         <code>stable</code>
       </td>
       <td>
-        <span className="sauceGreen">Active</span>
+        <span className="sauceGold">January 26th, 2026</span>
```

Comment:
You are just changing the banner, but not changing the content of this version. I think thats not really optimal to way of documentation. 

Instead of having just this large banner, we should make sure existing doc is correct and the banner just announce the change with a quick summary.